From e102663f5b7dd81813cbcc079615e1d208f83eca Mon Sep 17 00:00:00 2001
From: RuiX Wang <ruix.wang@intel.com>
Date: Thu, 24 May 2018 03:19:14 +0100
Subject: [PATCH 1928/2302] Revert "media:intel-ipu4:isys:v4l2-compliance fix
 for ISA subdev"

This reverts commit 78a2bc51c2b0248b9054efbda4e26ea0d804fab4.

Change-Id: I05fea6e9d7817a29bfeaf0cf18bc3f243d67632b
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/pci/intel/ipu4/ipu4-isys-isa.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/pci/intel/ipu4/ipu4-isys-isa.c b/drivers/media/pci/intel/ipu4/ipu4-isys-isa.c
index 11ed206..3bc6510 100644
--- a/drivers/media/pci/intel/ipu4/ipu4-isys-isa.c
+++ b/drivers/media/pci/intel/ipu4/ipu4-isys-isa.c
@@ -349,8 +349,8 @@ static void isa_set_ffmt(struct v4l2_subdev *sd,
 	case ISA_PAD_CONFIG:
 	case ISA_PAD_3A:
 		ffmt->code = MEDIA_BUS_FMT_FIXED;
-		ffmt->width = IPU_ISYS_MIN_WIDTH;
-		ffmt->height = IPU_ISYS_MIN_HEIGHT;
+		ffmt->width = 0;
+		ffmt->height = 0;
 		fmt->format = *ffmt;
 		return;
 	case ISA_PAD_SOURCE_SCALED: {
-- 
2.7.4

